spec/models/period_filter_spec.rb

```diff
@@ -92,4 +92,24 @@
       end
     end
   end
+
+  describe '#legal_periods' do
+    subject(:legal_periods) do
+      described_class.new.legal_periods
+    end
+
+    context 'when NO percents exist for employee and employer' do
+      it 'returns the periods with the defaults added in' do
+        expect(legal_periods.first.employee_percent).to eq(1)
+        expect(legal_periods.first.employer_percent).to eq(1)
+      end
+    end
+
+    context 'when percents exist for employee and employer' do
+      it 'returns the periods with the defaults added in' do
+        expect(legal_periods[1].employee_percent).to eq(3)
```

Comment:
I disagree with this because `.first` and `.last` are more readable but if I want to access any other element in an array, the bracket notation must be used. Would be good to get others views @tomas-stefano?